src/patches/vs1053b-patches-flac-latm.plg

```diff
@@ -0,0 +1,1142 @@
+/* User application code loading tables for VS10xx */
+#if 0
+void LoadUserCode(void) {
+  int i = 0;
+
+  while (i<sizeof(plugin)/sizeof(plugin[0])) {
+    unsigned short addr, n, val;
+    addr = plugin[i++];
+    n = plugin[i++];
+    if (n & 0x8000U) { /* RLE run, replicate n samples */
+      n &= 0x7FFF;
+      val = plugin[i++];
+      while (n--) {
+        WriteVS10xxRegister(addr, val);
+      }
+    } else {           /* Copy run, copy n samples */
+      while (n--) {
+        val = plugin[i++];
+        WriteVS10xxRegister(addr, val);
+      }
+    }
+  }
+}
+#endif
+
+#ifndef SKIP_PLUGIN_VARNAME
+const unsigned short patches-flac-latm[] = { /* Compressed plugin */
```

Comment:
A name `short patches-flac-latm` seem to be not valid. 
Using the `-` is rather not a valid variable/const name if about C++ style guide.
A snake-case or camel-case would be correct.